packages/bot-web-ui/src/components/dashboard/dashboard-components/index.scss

```diff
@@ -0,0 +1,178 @@
+.bot-dashboard {
+    .toolbar__section {
+        justify-content: end;
+    }
+    .run-panel__container {
+        height: calc(100vh - 5rem) !important;
+        position: unset !important;
+        transform: translateX(calc(350px));
+        transition: all 0.4s;
+        margin-top: 1rem;
+    }
+    .dc-drawer--open {
+        transform: translateX(calc(0px)) !important;
+    }
+}
+.dashboard {
+    &__container {
+        border: none;
+        max-width: 100%;
+        background: var(--general-main-1);
+        width: 100%;
+
+        .dc-tabs {
+            &__active-line {
+                background-color: var(--general-main-1);
+                transition: none;
+            }
+
+            &__active {
+                background: var(--general-main-1);
+            }
+
+            &__list {
+                background: var(--general-section-6);
+                justify-content: start;
+            }
+
+            &__item {
+                border-radius: 1rem 1rem 0px 0px;
```

Comment:
or maybe border-radius: 1rem 0px;  ?

Reply:
We only need border radius for top-left and top-right so we did this..
but I think
```
border-top-left-radius: 1rem;
border-top-right-radius: 1rem;
```
it will make more sense.

✅